README.md

```diff
@@ -1,40 +1,41 @@
 # Welcome to the lab on enabling GitHub Pages!
 
-- [About the codebase](#about-the-codebase)
-- [Lab instructions](#lab-instructions)
-- [Grading criteria](#grading-criteria)
-- [A note about usage of GitHub Actions](#a-note-about-usage-of-github-actions)
-- [References](#references)
+This lab is an exercise to check your knowledge on enabling GitHub Pages. It is automatically graded via a workflow once you have completed the instructions.
 
+**Quick links:**
+- [About this lab](#about-this-lab)
+- [Instructions](#instructions)
+- [Seeing your result](#seeing-your-result)
+- [Troubleshooting](#troubleshooting)
+- [Useful resources](#useful-resources)
 
-## About the codebase
-The codebase contains a starter GitHub pages site inside the `docs/` directory, but you can use any content you'd like as long as the conditions specified in the [grading criteria](#grading-criteria) are met. 
+## About this lab
 
-A grading script and a setup utility exist inside of the `.github/` directory. You do not need to use these workflows for any purpose and altering their contents will affect the repository's ability to assess your lab and give feedback.
+:warning: A grading script and a setup utility exist inside of the `.github/` directory. You do not need to use these workflows for any purpose and **altering their contents will affect the repository's ability to assess your lab and give feedback.**
 
-## Lab instructions
+:information_source: This lab utilizes [GitHub Actions]( https://docs.github.com/en/actions) to grade the exercise, which means that it may take the grading workflow a few seconds and sometimes minutes to run.
```

Comment:
Thanks for this! I think the emoji helps break up the wall of text quite a bit.

Reply:
Yes, it's also meant to attract the learner's eye as we want them to read this info and not just skip it.